src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
please consider renaming valiable 'ballColor' to 'color' and 'ballNumber' to 'Number'